modules/cnf-about-topology-aware-lifecycle-operator-config.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: CONCEPT
+[id="cnf-about-topology-aware-lifecycle-operator-config_{context}"]
+= About the Topology Aware Lifecycle Operator configuration
+
+The Topology Aware Lifecycle Operator manages the deployment of {rh-rhacm-first} policies for one or more {product-title} clusters. TALO allows you to control the following actions:
+
+* The timing of the update
+* The number of {rh-rhacm}-managed clusters
+* The update order of the clusters
+
+TALO supports the orchestration of the {product-title} y-stream and z-stream updates, and day-two operations on y-stream and z-streams.
+
+[id="cnf-about-topology-aware-lifecycle-operator-about-policies_{context}"]
+== About managed policies used with Topology Aware Lifecycle Operator
+
+TALO uses {rh-rhacm} policies for cluster updates.
```

Comment:
All this section below is repeated in the file below. Is this intended?

Reply:
No, it was an accident on my part while restructuring the docs. This section will be in a separate file.